src/main/java/rx/Single.java

```diff
@@ -1789,4 +1790,18 @@ public void onNext(T t) {
         return zip(this, other, zipFunction);
     }
 
+    /**
+     * Converts a Single into a {@link BlockingSingle} (a Single with blocking operators).
+     * <dl>
+     *  <dt><b>Scheduler:</b></dt>
+     *  <dd>{@code toBlocking} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     *
+     * @return a {@code BlockingSingle} version of this Single.
+     * @see <a href="http://reactivex.io/documentation/operators/to.html">ReactiveX operators documentation: To</a>
+     */
+    public final BlockingSingle<T> toBlocking() {
```

Comment:
This should start out as experimental unless the RxJava contributors want to fast-track this.